docs/html/pk-using.html

```diff
@@ -70,9 +70,8 @@ <h2>Using graphical tools:</h2>
 <code>gnome-software</code> and <code>gnome-packagekit</code> provide a rich
 set of GTK tools for automatically updating your computer and installing software.
 </p>
-<a href="http://kde-apps.org/content/show.php/Apper?content=84745">
-Apper</a> is the name of the KDE graphical tools designed
-for PackageKit.
+<a href="https://apps.kde.org/discover/">
+Discover</a> is the name of the KDE interface for PackageKit.
```

Comment:
KDE Plasma interface?

Reply:
I used the same terminology as https://github.com/PackageKit/PackageKit/pull/515